e2e/tests/migration_validation_fixture.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Morse account validation
+
+  Background: Prepare MsgImportMorseClaimableAccounts JSON
+    Given the user has the pocketd binary installed
+    And a valid MsgImportMorseClaimableAccounts JSON exists with the following morse account state:
+      | index | unstaked_balance | application_stake | supplier_stake |
```

Comment:
```suggestion
      | account_index | unstaked_balance | application_stake | supplier_stake |
```

Reply:
Gonna switch to `morse_src_address`: https://github.com/pokt-network/poktroll/pull/1178#discussion_r2039320585